Formula/htmltest.rb

```diff
@@ -0,0 +1,18 @@
+class Htmltest < Formula
+  desc "HTML validator written in Go"
+  homepage "https://github.com/wjdp/htmltest"
+  url "https://github.com/wjdp/htmltest/archive/v0.14.0.tar.gz"
+  sha256 "add922cf1dd957afba2927d401184c1d2331983a6d8ed96dd10f5001930cebf8"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    # executables being tested: `system "#{bin}/program", "do", "something"`.
+    system "#{bin}/htmltest", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Done! Thanks for the feedback, I missed that line in the docs.